backend/pkg/sqlmanager/mysql/mysql-manager.go

```diff
@@ -350,7 +350,7 @@ func buildTableCol(record *buildTableColRequest) string {
 	}
 
 	if record.ColumnDefault != "" {
-		pieces = append(pieces, fmt.Sprintf("DEFAULT (%s)", record.ColumnDefault))
+		pieces = append(pieces, fmt.Sprintf("DEFAULT '%s'", record.ColumnDefault))
```

Comment:
Hi! Thanks for working on this bug. This definitely fixes default values with slashes but will break many other default values like CURRENT_TIMESTAMP and NOW() etc.. Need more logic here to handle these other cases

Reply:
Hi,
In this commit ( https://github.com/nucleuscloud/neosync/pull/2774/commits/34f020ae58533f3b105c6a301e4a46dbe49b9095 ) we tested several scenarios and noticed that the "extra" column always contains a value when the "default" column is a function. This value never appears when the "default" column is a string.

Based on this observation, we defined a new type called "ColumnDefaultType." It is set as a string when the column's default value contains information, and the "extra" column does not.

We then add quotes around the value if it’s a string, otherwise, we add parentheses as previously done.

Finally, if "ColumnDefaultType" is empty, nothing is done, just like before.

Tests were conducted on tables with both string default values and default values like now(). Everything works as expected.

This has resolved our issue while preserving the correct functionality when a function or other type is present.

Looking forward to your feedback.